sync/patches/values/patch.sh

```diff
@@ -14,5 +14,6 @@ readonly script_dir_rel=".${script_dir#"${repo_dir}"}"
 set -x
 git apply "${script_dir_rel}/000-values.patch"
 git apply "${script_dir_rel}/000-values.schema.patch"
+git apply "${script_dir_rel}/001-add-monitoring.patch"
```

Comment:
What is this for? You probably don't need this since you're not modifying the upstream chart